src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -9,30 +9,33 @@
 			class="tw-mb-1 tw-rounded" :style="{width: '100%', height: '15.75rem'}"
 		/>
 
-		<!-- TODO: Once click interaction for this is in place, we need to disable the click
-		if allSharesReserved is true
-		ie.
-		:click="`${allSharesReserved ? 'correct redirect' : ''}`" -->
-		<borrower-image
-			v-if="!isLoading"
-			class="
-				tw-w-full
-				tw-bg-black
-				tw-rounded
-			"
-			:alt="'photo of ' + borrowerName"
-			:aspect-ratio="3 / 4"
-			:default-image="{ width: 336 }"
-			:hash="imageHash"
-			:images="[
-				{ width: 336, viewSize: 1024 },
-				{ width: 336, viewSize: 768 },
-				{ width: 416, viewSize: 480 },
-				{ width: 374, viewSize: 414 },
-				{ width: 335, viewSize: 375 },
-				{ width: 280 },
-			]"
-		/>
+		<!-- If allSharesReserved, disable link by make it a span -->
```

Comment:
Why do we need to remove this link at all?

Reply:
The requested allSharesReserved state from Nathan is "disabled button, (no click targets) - and funding complete". 

I mentioned that it felt odd to block users from learning more about a borrower. But he wants to keep this block in place until we get the recommended loan cards on funded borrower pages.

Reply:
Conversation discussed and resolved in Slack.